assets/src/edit-story/utils/useDoubleClick.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect, useState } from '@wordpress/element';
+
+const useDoubleClick = ( onClick, onDoubleClick ) => {
```

Comment:
There's a discussion in #3800 re: the editing condition triggering. If we consider the triggering to be based on a single click (or Enter key) on a selected element, we may not need a special double-click handling?

Reply:
Yes, I removed the double-click usage from the codebase, but (for now) kept the util hook. If image editing is also to be made without double-click, I'll remove it completely.

Reply:
Image editing is still done with double-click in the UX and thus stays. I have added documentation.